examples/twitter_inlet.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+import tweepy
+from databay import Inlet, Link, Record
+from databay.outlets import PrintOutlet
+from databay.planners import ApsPlanner, SchedulePlanner
```

Comment:
I know you added this upon my suggestion during the call, tho now we probably only need to import one of these.